### PR TITLE
`<Calendar/>` - Add `autoFocus` prop

### PR DESCRIPTION
**Pull request by hepiyellow**
Workaround for this Issue
https://github.com/wix/wix-style-react/issues/2799

`autoFocus` is enabled by default.
when disabled, then the above issue should not occur.

### CODE REVIEW

File: src/Calendar/Calendar.js [hunk 1]
```diff
@@ -328,22 +328,40 @@ export default class Calendar extends WixComponent {
...
         onClick={this._preventActionEventDefault}
       >
         <DayPicker
-          ref={this._focusSelectedDay}
+          ref={ref => (this.dayPickerRef = ref)}
           {...this._createDayPickerProps()}
         />
       </div>
     );
+    // console.log('result= ', result);
```
Comment (yardnsm) [inline src/Calendar/Calendar.js]: who dis

File: stories/Popover/index.story.js [hunk 2]
```diff
@@ -115,27 +129,27 @@ export default {
...
-      <Example title="Flip behaviour">
-        <ExampleFlip />
-      </Example>
-
-      <Example title="Fixed behaviour">
-        <ExampleFixed />
-      </Example>
-    </div>
-  ),
+  // examples: (
```
Comment (yardnsm) [inline stories/Popover/index.story.js]: Uncomment

File: src/Calendar/Calendar.js (hunk 1 above)
Comment (hepiyellow) [reply src/Calendar/Calendar.js]: Yayks !!!!!!!!!!!!!!!!!!!

**Comment by hepiyellow** [comment]
Replaced by #2838 due to push force (to avoid Applitools baseline mixup)

**Pull request closed without merging**